### ISSUE TITLE
Refineries keep running although grid is concealed

### ISSUE THREAD

**Issue by LordTylus** [TorchAPI/Concealment#21]
Hello there,

fowlplaychiken\Mordith found out about a not so fun issue with Concealment. If a player is out of range the grids stops updating and the log message of grids being concealed appears as normal.

However a Refinery that still has stuff in its inventory seems to keep being updated and can actually produce stuff.

I was able to verify it. Its interesting how they were able to do that since power is not updated, but I assume refineries just keep working until the power management system says otherwise and since it doesnt say anything due to lack of updates the refinery seems to be fine.

This could explain why the simspeed dropped by a good amount compared to beforee the last Major update.

**Comment by Fowlplaychiken** [comment]
I found this initially, with help from several players in server. We tested every combination (player in cryo offline, player nearby w/ helmet off and offline, and player "dead' and offline). In all tests, refinery continued refining even when grid was in concealment. Log shows concealment working as intended. Guardians server.

**Comment by Fowlplaychiken** [comment]
Just tested and confirmed - refinery able to pull stone from cargo via conveyor when in 'concealment' as well....

**Comment by smooreace** [comment]
I think its always done that

**Comment by LordTylus** [comment]
No I am very sure it did not. 

When I played on our server some time ago (like a few months) I left a refinery full of stuff and logged off for a weekend. When I came back the refinary didnt manage to refine anything in that time. 

So it must have worked differently before.

**Comment by smooreace** [comment]
I have the opposite... I moved my server to a new machine, did a ton of
mining one evening, and logged off. Came back the next day to find nothing
had been processed. Days later I found that the owner of the new machine
had checked the box named something like "Pause server when empty". Once
that was unchecked, all of out processing, both refineries and assemblers,
continued to work even when no-one was there.
